config/environments/development.rb

```diff
@@ -10,59 +9,72 @@
+      'Cache-Control' => "public, max-age=#{2.days.to_i}"
+    }
+  else
+    config.action_controller.perform_caching = false
+
+    config.cache_store = :null_store
+  end
+
+  # Store uploaded files on the local file system (see config/storage.yml for options).
+  config.active_storage.service = :local
+
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
 
-  ## Config for MailCatcher ##
-  # Install mailcatcher locally on your machine 'gem install mailcatcher'
-  # Run 'mailcatcher' in the terminal to start the server
-  # Open 'http://localhost:1080/' in your browser to see mail sent
-  config.action_mailer.delivery_method = :smtp
-  config.action_mailer.smtp_settings = { :address => "localhost", :port => 1025 }
+  config.action_mailer.perform_caching = false
 
   # Print deprecation notices to the Rails logger.
   config.active_support.deprecation = :log
 
   # Raise an error on page load if there are pending migrations.
   config.active_record.migration_error = :page_load
 
+  # Highlight code that triggered database queries in logs.
+  config.active_record.verbose_query_logs = true
+
   # Debug mode disables concatenation and preprocessing of assets.
   # This option may cause significant delays in view rendering with a large
   # number of complex assets.
   config.assets.debug = true
 
-  # Adds additional error checking when serving assets at runtime.
-  # Checks for improperly declared sprockets dependencies.
-  # Raises helpful error messages.
-  config.assets.raise_runtime_errors = true
-  
-  # http://pontiiif.brumfieldlabs.com/api/v0.0/search/Irish
-  config.pontiiif_server = 'http://pontiiif.brumfieldlabs.com/'
-  
-  Ahoy.geocode = false
+  # Suppress logger output for asset requests.
+  config.assets.quiet = true
 
-  # Raises error for missing translations
+  # Raises error for missing translations.
   # config.action_view.raise_on_missing_translations = true
 
+  # Use an evented file watcher to asynchronously detect changes in source code,
+  # routes, locales, etc. This feature depends on the listen gem.
+  config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  Rails.application.routes.default_url_options[:host] = 'localhost:3000'
+
+  config.action_mailer.default_url_options =  { host: 'localhost:3000' }
+
   # location of system calls on this machine
   NEATO = '/usr/bin/neato'
   RAKE = '/usr/bin/env rake'
-  TEX_PATH='/usr/local/texlive/2017/bin/x86_64-linux/'  
+  TEX_PATH='/usr/local/texlive/2017/bin/x86_64-linux/'
+
+  config.pontiiif_server = 'http://pontiiif.brumfieldlabs.com/'
 
+  ## Config for MailCatcher ##
+  # Install mailcatcher locally on your machine 'gem install mailcatcher'
+  # Run 'mailcatcher' in the terminal to start the server
+  # Open 'http://localhost:1080/' in your browser to see mail sent
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = { :address => "localhost", :port => 1025 }
```

Comment:
let's make sure all the action_mailer configs are next to each other.  (There's an action_mailer cache setting somewhere higher in this file.)

Reply:
@saracarl done, thanks.